app/controllers/tokens_controller.rb

```diff
@@ -1,10 +1,11 @@
-class TokensController < ApplicationController
+
+ class TokensController < ApplicationController
   before_action :set_token, only: [:validate]
   skip_before_action :authenticate_user, only: [:validate]
 
   def validate
     @user = @token.user
-    @user.update(active: true)
+    render json: {}, status: 200 if @user.update(active: true)
```

Comment:
there's a jBuilder view that renders the json object for this, I guess. calling `render json: {}` just renders an empty object, whereas it ought to return the user's detail with token